tests/general/LineCountBearTest.py

```diff
@@ -16,12 +16,26 @@ def setUp(self):
     def test_run(self):
         self.section.append(Setting('max_lines_per_file', 1))
         self.check_results(
-            self.uut, ['line 1', 'line 2', 'line 3'],
+            self.uut, ['line 1', 'line 2', 'line 3', '\n'],
             [Result.from_values('LineCountBear',
-                                'This file had 3 lines, which is 2 lines more '
+                                'This file had 4 lines, which is 3 lines more '
                                 'than the maximum limit specified.',
                                 severity=RESULT_SEVERITY.NORMAL,
                                 file='default')],
             filename='default')
         self.check_validity(self.uut, ['1 line'])
         self.check_validity(self.uut, [])  # Empty file
+
+    def test_exclude_blank_lines(self):
+        self.section.append(Setting('exclude_blank_lines', True))
+        self.section.append(Setting('max_lines_per_file', 2))
+        self.check_results(
+            self.uut, ['line 1', ' ', 'line 2',
```

Comment:
I would suggest you to add all the possible cases, ie. 
* leading whitespaces/tabs before the `line x` -> '   line x'
* trailing whitespaces/tabs after the `line x` -> 'line x   '
* both of the above cases `line x` -> '   line x   '
* some lines with multiple tabs